pyproject.toml

```diff
@@ -49,3 +39,6 @@ update_changelog_on_bump = true
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.mypy]
+exclude = ["ds_caselaw_utils/courts_schema_types_autogenerated.py"]
```

Comment:
If we globally exclude this in pre-commit config, do we need this line?

Reply:
Needed a lot of [necessary] tidying to remove it, but yes